asv_bench/asv.conf.dask.json

```diff
@@ -13,31 +13,9 @@
     // project being benchmarked
     "repo": "..",
 
-    // The Python project's subdirectory in your repo.  If missing or
```

Comment:
Due to the simplicity of the changes, I also want to remove unnecessary comments in the configs, in **this** pull request.